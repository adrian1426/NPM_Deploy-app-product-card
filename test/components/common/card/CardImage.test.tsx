import React from 'react';
import renderer from 'react-test-renderer';
import CardImage from '../../../../src/components/common/card/CardImage';

describe('CardImage', () => {
  test('should render CardImage', () => {
    const wrapper = renderer.create(
      <CardImage />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});