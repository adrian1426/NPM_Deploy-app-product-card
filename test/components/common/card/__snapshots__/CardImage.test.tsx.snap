// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImage should render CardImage 1`] = `
<img
  className="productImg undefined"
/>
`;
